### DISCUSSION TITLE
Understanding Kubernetes' Cluster Networking | George Aristy

### DISCUSSION THREAD

**Discussion by giscus** [llorllale/llorllale.github.io#56] [Announcements]
# Understanding Kubernetes' Cluster Networking | George Aristy

Kubernetes is a system for automating deployment, scaling, and management of containerized applications. Networking is a central part of Kubernetes, and in this article we will explore how Kubernetes configures the cluster to handle east-west traffic. We’ll reserve discussion on north-south traffic for a later article.

https://llorllale.github.io/posts/k8s-cluster-network/

**Comment by Veronica4036** [comment]
The most Amazing and detailed blog on Cluster Networking I have read till now!

**Reply by llorllale** [reply]
I'm happy you liked it.
